Document the -native-compiler option (#13684)
coq/coq: doc/sphinx/proofs/writing-proofs/rewriting.rst

   in :cite:`FullReduction`. If Coq is running in native code, it can be
   typically two to five times faster than :tacn:`vm_compute`. Note however that the
   compilation cost is higher, so it is worth using only for intensive

>    computations. According to the configuration, this tactic can default to

Reviewer: ```suggestion
   computations. With this configuration, this tactic can default to
```

"According to the configuration" doesn't sound very good.  Not sure if my suggestion means what you intend.

Reviewer 2: Not quite. "Depending on" would fit better the intended meaning. I guess "either" should also be added before "default".

>    :tacn:`vm_compute`, recompile a lot of dependencies or fail due to some missing

Reviewer 2: I would just remove "a lot of".

>    dependency precompilation,

Reviewer 2: "precompiled dependencies" seems better.
